lib/engine.js

```diff
@@ -55,6 +65,21 @@ Engine.prototype.start = function (callback) {
     this._config.storage.mongoOpts,
     { logger: log }
   );
+  this.networkQueue = new NetworkMessageQueue({
+    connection: {
+      url: `amqp://${USER}:${PASS}@${HOST}:${PORT}`,
+    },
+    exchange: {
+      name: 'exchangeName',
+      type: 'direct',
+    },
+    queue: {
+      name: QUEUE_NAME,
+    },
+    routingKey: {
+      name: 'routingKeyName',
+    },
+  });
```

Comment:
(same here) Unsure if it's ok to import them right here or to create a file to import them